source/patterns/@aws-solutions-constructs/aws-kinesisstreams-gluejob/README.md

```diff
@@ -85,30 +85,30 @@ _Parameters_
-| fieldSchema?        | [`CfnTable.ColumnProperty[]`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnTable.ColumnProperty.html) | Either one of @existingTable, @tableProps or @fieldSchema has to be provided. If none are provided then the construct will throw an error                                                                                                                                                                                            |
-| outputDataStore?    | [`SinkDataStoreProps`](#sinkdatastoreprops)                                                                                   | The output datastore properties where the ETL output should be. Current datastore types suported is only S3.                                                                                                                                                                                                                         |
+| **Name**            | **Type**                                                                                                                      | **Description**                                                                                                  |
+| :------------------ | :---------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| kinesisStreamProps? | [`kinesis.StreamProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-kinesis.StreamProps.html)                | Optional user-provided props to override the default props for the Kinesis stream.                               |
+| existingStreamObj?  | [`kinesis.Stream`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-kinesis.Stream.html)                          | Existing instance of Kinesis Stream, if this is set then kinesisStreamProps is ignored.                          |
+| glueJobProps?       | [`cfnJob.CfnJobProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnJobProps.html)                    | User provided props to override the default props for the AWS Glue Job.                                          |
+| existingGlueJob?    | [`cfnJob.CfnJob`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-glue.CfnJob.html)                              | Existing Existing instance of AWS Glue Job, if this is set then glueJobProps is ignored.                         |
```

Comment:
I'm getting a little lost in the green/red adds/deletions, but it looks like "Existing" is repeated here?

Reply:
Fixed.